src/renderer/App.tsx

```diff
@@ -1,13 +1,15 @@
-import Main from 'electron/main';
+// import Main from 'electron/main';
 import { MemoryRouter as Router, Routes, Route } from 'react-router-dom';
 import './App.css';
 import LoginPage from './pages/LoginPage';
+import TabsComponent from './pages/TabsComponent';
 
 export default function App() {
   return (
     <Router>
       <Routes>
         <Route path="/login" element={<LoginPage />} />
+        <Route path="/tabs" element={<TabsComponent />} />
```

Comment:
This path should be something like `/master` and then a new `MasterPage` that just renders your `Tabs` component for now